nautobot/core/templates/inc/nav_menu.html

```diff
@@ -2,27 +2,57 @@
 {% load registry %}
 {% load helpers %}
 {% registry %}
-<nav class="navbar navbar-default navbar-fixed-top">
+<nav class="navbar navbar-default navbar-fixed-left navbar-inverse">
     <div class="container-fluid">
         <div class="navbar-header">
-            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
-                <span class="sr-only">Toggle navigation</span>
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-            </button>
             <a class="navbar-brand" href="{% url 'home' %}">
                 <img src="{% custom_branding_or_static 'logo' 'img/nautobot_logo.svg' %}" height="30" />
             </a>
         </div>
 
-        <div id="navbar" class="navbar-collapse collapse">
+        <div id="navbar">
+            {% if request.user.is_authenticated or not "HIDE_RESTRICTED_UI"|settings_or_config %}
```

Comment:
We shouldn't expose the search form if the user isn't logged in should we?

```suggestion
            {% if request.user.is_authenticated %}
```

Reply:
IDK, I just left this condition intact. To give it some more context - it was implemented in #1857